This patch is required to properly recognize the Windows-specific batch script
wrappers bundled with the conda-forge version of Graphviz. See also:
    https://github.com/conda-forge/graphviz-feedstock/blob/bbfe3c7be2a448dd11db165a18c3edfd5ee6a95d/recipe/bld.bat#L27-L32

diff -Naur pydot/pydot.py pydot-patched/pydot.py
--- pydot/pydot.py	2017-12-25 11:41:43.000000000 -0500
+++ pydot-patched/pydot.py	2018-08-11 00:10:23.391342812 -0400
@@ -1828,8 +1828,8 @@
         else:
             args = []
         if os.name == 'nt' and prog in default_names:
-            if not prog.endswith('.exe'):
-                prog += '.exe'
+            if not os.path.splitext(prog)[1]:
+                prog += '.bat'
         # temp file
         tmp_fd, tmp_name = tempfile.mkstemp()
         os.close(tmp_fd)
